dropletutils-read-10x-counts.R

```diff
@@ -13,14 +13,47 @@ option_list = list(
     action = "store",
     default = NA,
     type = 'character',
-    help = "A comma-separated list containing one or more directory names, each corresponding to a 10X sample. Each directory should contain the 'matrix.mtx', 'genes.tsv' and 'barcodes.tsv' files generated by CellRanger. Alternatively, strings may contain a path to a HDF5 file in the sparse matrix format generated by 10X. These can be mixed with directory names when type='auto'."
+    help = "A comma-separated list containing one or more directory names, each
+            corresponding to a 10X sample. Each directory should contain the 'matrix.mtx',
+            'genes.tsv' and 'barcodes.tsv' files generated by CellRanger. 
+            Alternatively, strings may contain a path to a HDF5 file in the sparse
+            matrix format generated by 10X. These can be mixed with directory 
+            names when type='auto'."
   ),
   make_option(
-    c("-c", "--col-names"),
+    c("-n", "--col-names"),
     action = "store",
     default = FALSE,
     type = 'logical',
-    help = "A logical scalar indicating whether the columns of the output object should be named with the cell barcodes."
+    help = "A logical scalar indicating whether the columns of the output object
+            should be named with the cell barcodes."
+  ),
+  make_option(
+    c("-m", "--metadata-files"),
+    action = "store",
+    default = NA, 
+    type = 'character',
+    help = "A comma-separated list containing one or more paths to the SDRF 
```

Comment:
Please remove SDRF references, it's too Atlas-specific. A simple tab-delimited file is all that's required.